setup.py

```diff
@@ -4,10 +4,10 @@
     long_description = fh.read()
 
 setup(
```

Comment:
none of the required test dependencies are specified in `setup.py`. Add these under the "tests" section: 

```
"flake8>=3.7.9,<4",
"isort>=5.8.0,<6",
"mypy>=0.910,<1",
 "pytest-cov",
"pytest-env",
"pytest-html",
"pytest-mock",
"pytest-timeout",
"pytest>=6.2.3,<7",
```